zerver/models.py

```diff
@@ -293,10 +293,8 @@ class Realm(models.Model):
     JITSI_MEET = u"Jitsi"
     GOOGLE_HANGOUTS = u"Google Hangouts"
     ZOOM = u"Zoom"
-    MAX_VIDEO_CHAT_PROVIDER_LENGTH = 40
     VIDEO_CHAT_PROVIDERS = [JITSI_MEET, GOOGLE_HANGOUTS, ZOOM]
```

Comment:
I would consider changing this data structure to a dictionary mapping between integers and strings; I think the result will be a lot cleaner code (the `indexOf` stuff is pretty messy).